azerothcore/mod-individual-xp
# Adding support for DB Assembler

Adding support for DB Assembler so the .sql files can be imported directly using the Import Custom function of db_assembler

How to test this:
pull the pull request in the modules folder
run db_assembler and use the `8` option
You should see in the console
```
Searching on /e/azerothcore/modules/mod-individual-xp/sql/characters/ ...
Searching on /e/azerothcore/modules/mod-individual-xp/sql/world/ ...
```

That means this is working as intended

Thanks @milestorme for the suggestion!
